content/en/synthetics/uptime_check.md

```diff
@@ -141,6 +141,24 @@ A check is considered `FAILED` if it doesn't satisfy the assertions configured f
 | UNABLE_TO_GET_ISSUER_CERT_LOCALLY  | The issuer certificate of a locally found certificate is not found. This usually means that the list of trusted certificates is not complete.                         |
 | UNABLE_TO_VERIFY_LEAF_SIGNATURE    | No signatures are verified because the certificate chain contains only one certificate, which is not self-signed, and the issuer is not trusted.                      |
 
+## External metrics
```

Comment:
I'd put this below a `Data collected`, or a `Metrics` header, just like we do for our integrations page, rather than `External metrics`. I used this term because in the internal doc, it was opposed to the internal metrics that were generated for internal troubleshooting :)